### PR TITLE
fix: Update Colors for the MarkdownInputBarView and other elements - ACC - 227

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we updated parts of the InputBarView such as:

- Markdown Button and MarkdownInputBarView, PopUpIconButtonView
- Send Button
- Typing Indicator View

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Markdown.swift [hunk 1]
```diff
@@ -25,20 +25,34 @@ extension ConversationInputBarViewController {
...
         }
 
         markdownButton.setIconColor(color, for: .normal)
+        markdownButton.setBorderColor(borderColor, for: .normal)
+        markdownButton.setBackgroundImageColor(backgroundColor, for: .normal)
+
+        markdownButton.setIconColor(SemanticColors.Button.textInputBarItemHighlighted, for: .highlighted)
+        markdownButton.setBorderColor(SemanticColors.Button.borderInputBarItemHighlighted, for: .highlighted)
+        markdownButton.setBackgroundImageColor(SemanticColors.Button.backgroundInputBarItemHighlighted, for: .highlighted)
+
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Markdown.swift:55]: Swift lint issue

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBar.swift [hunk 2]
```diff
@@ -134,14 +134,14 @@ final class InputBar: UIView {
     let markdownView = MarkdownBarView()
 
     var editingBackgroundColor = SemanticColors.LegacyColors.brightYellow
-    var barBackgroundColor: UIColor? = UIColor.from(scheme: .barBackground)
-    var writingSeparatorColor: UIColor? = .from(scheme: .separator)
+    var barBackgroundColor: UIColor?  = SemanticColors.SearchBar.backgroundInputView
+    var writingSeparatorColor: UIColor?  = SemanticColors.View.backgroundSeparatorCell
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBar.swift:138]:
```suggestion
    var writingSeparatorColor: UIColor? = SemanticColors.View.backgroundSeparatorCell
```

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift [hunk 3]
```diff
@@ -962,8 +966,8 @@ extension ConversationInputBarViewController: UIGestureRecognizerDelegate {
 
             markdownButton.centerXAnchor.constraint(equalTo: markdownButton.superview!.centerXAnchor),
             markdownButton.bottomAnchor.constraint(equalTo: markdownButton.superview!.bottomAnchor, constant: -14),
-            markdownButton.widthAnchor.constraint(equalToConstant: senderDiameter),
-            markdownButton.heightAnchor.constraint(equalToConstant: senderDiameter),
+            markdownButton.widthAnchor.constraint(equalToConstant: 40),
+            markdownButton.heightAnchor.constraint(equalToConstant: 32),
```
[the remark this answers is not in the document]
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift:970]: They're not magic numbers. That's the values I got from the Sigma file

**Review by johnxnguyen — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift (hunk 3 above)
[the remark this answers is not in the document]
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift:970]: It is not clear these numbers are the height and width of what

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Markdown.swift (hunk 1 above)
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Markdown.swift:55]: There is a swift lint issue

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift (hunk 3 above)
[the remark this answers is not in the document]
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift:970]: Done!

**Review by KaterinaWire — APPROVED** [review]

**Pull request merged**